TextToSpeech/TextToSpeechImplementation.cpp

```diff
@@ -62,6 +65,17 @@ namespace Plugin {
         JsonObject config;
         config.FromString(service->ConfigLine());
 
+        InputValidation::Instance().addValidator("ttsendpoint", ExpectedValues<std::string>("^https?://[a-zA-Z0-9]+.*"));
+        InputValidation::Instance().addValidator("ttsendpointsecured", ExpectedValues<std::string>("^https://[a-zA-Z0-9]+.*"));
+        InputValidation::Instance().addValidator("double_str", ExpectedValues<std::string>("^-?[0-9]+(\\.[0-9]+)?"));
+        InputValidation::Instance().addValidator("speechid", ExpectedValues<uint32_t>(1, UINT32_MAX));
+        InputValidation::Instance().addValidator("text", ExpectedValues<std::string>("^[a-zA-Z0-9]+"));
```

Comment:
I am little hesitant about validating the text input, the input could have underscores, question marks or even other characters which are not ascii. We can't throw error on such cases. Is it imperative to validate text?

Reply:
I think text validator is not necessary in our cases. I think we can remove this.